src/features/identityCheck/pages/phoneValidation/__tests__/SetPhoneNumber.native.test.tsx

```diff
@@ -23,5 +25,36 @@ describe('SetPhoneNumber', () => {
     await waitFor(() => expect(getByText("J'ai compris")).toBeTruthy())
   })
 
+  describe('continue button', () => {
+    it('should enable the button when the phone number is valid', async () => {
+      const { getByTestId, getByPlaceholderText } = render(<SetPhoneNumber />)
+      const button = getByTestId('Continuer')
+
+      await waitFor(() => expect(button.props.style.backgroundColor).toEqual(ColorsEnum.GREY_LIGHT))
+
+      const input = getByPlaceholderText('06 12 34 56 78')
+      fireEvent.changeText(input, '612345678')
+
+      expect(button.props.style.backgroundColor).toEqual(ColorsEnum.PRIMARY)
```

Comment:
Je sais que tu as reproduit ce qui était dans l'existant donc c'est plutôt une réflexion d'amélioration : 

Est-ce qu'on n'y gagnerait pas à tester la désactivation du bouton sur une vraie valeur de désactivation, plutôt que sur une donnée de couleur ? 

J'essaie de regarder comment faire, c'est pas si simple avec React Native à première vue

Reply:
Je m'étais notée de demander à Lucas qui avait entamé une réflexion à ce sujet, mais apparemment une màj de RTL pourrait débloquer la situation (cf discussion slack entre yorick et moi)